packages/core/src/transformFeedback/TransformFeedback.ts

```diff
@@ -0,0 +1,29 @@
+import { Runner } from '@pixi/runner';
+import type { Buffer } from 'pixi.js';
```

Comment:
Please import Buffer locally not from `pixi.js`.
```suggestion
import type { Buffer } from '../geometry/Buffer';
```